config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "broker_host":"localhost:9092",
+  "broker_host":"as01.hadoop:9092",
```

Comment:
Maybe we should hide this. I understand this aids the development process so my suggestion would be to add `config.json` in `.gitignore` and just provide a `config.json.template` in the codebase as an example .